docs/src/backwards-compatibility.md

```diff
@@ -3,15 +3,18 @@ title: Backward Compatibility Policy
 ---
 
 As the Solana developer ecosystem grows, so does the need for clear expectations around
-breaking API and behavior changes affecting applications and tooling built for Solana.
+breaking API and behavior changes affecting applications and tooling built for Solana by Anza.
 In a perfect world, Solana development could continue at a very fast pace without ever
 causing issues for existing developers. However, some compromises will need to be made
-and so this document attempts to clarify and codify the process for new releases.
+and so this document attempts to clarify and codify the process for new releases. Furthermore,
+there will be a growing number of validator clients maintained seperately by distinct teams. 
+Coordinating across these teams to ensure the reliability of the network will require ongoing
+communication. 
 
 ### Expectations
 
-- Solana software releases include APIs, SDKs, and CLI tooling (with a few [exceptions](#exceptions)).
-- Solana software releases follow semantic versioning, more details below.
+- Agave software releases include APIs, SDKs, and CLI tooling (with a few [exceptions](#exceptions)).
```

Comment:
We will probably decouple these soon, but for now this is still accurate.

There's nothing in the exceptions section so we can get rid of that.